File: 0001_0100/0043/0043.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    m, t = 0, 0
    for x in f.readline():
        if x == '0':
            t += 1
        else:
            if t > m:
                m = t;
            t = 0
    q.write(str(m))

File: 0001_0100/0043/0043.go
package main

import (
	"fmt"
	"os"
	"strings"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var s string
	fmt.Fscan(f, &s)
	n := 0
	for _, v := range strings.Split(s, "1") {
		if len(v) > n {
			n = len(v)
		}
	}
	fmt.Fprint(q, n)
}
